walkthrough/meadow.md

```diff
@@ -10,3 +10,5 @@ Here's a summary of this walkthrough, you don't have to manually execute anythin
 2. **Add the new meadow step to the dag**, including its dependencies.
 
 3. **Run `make test` in `etl`** and  ensure the step runs well. To run the step: `etl data://meadow/example_institution/YYYY-MM-DD/example_dataset`
+
+`meadow` is for _syntactic harmonisation_, meaning getting something into the right format. The idea is that instead of sitting on the dataset that you're working on for a long time without sharing it, you could spend a few hours and get it into a form where others can already reuse it. Out of this you get more, smaller pull-requests. It's also important to have this form for debugging, so that we can compare changes we make later to the "original data".
```

Comment:
I don't know if the sentence `The idea ...` is very realistic. So far I have never pushed just the meadow steps, because I often need to rethink them (e.g. add or remove raw data, or create one or several tables) while working on the garden steps. I would say that the goal of `meadow` is to do the minimum possible work on the original data, to convert it into one (or more than one) table.

Reply:
That's actually Lars's sentence :D. I shamelessly stole it from one of our discussions on github. I guess it's up to everyone's workflow. Ideally we'd be creating small incremental PRs and get them reviewed continuously, but honestly that adds so much overhead that it's easier to have one monstrous PR (and someone who's willing to review it). I do the same...